phantomgames/GameDetails/GameDetailViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  GameDetailsViewController.swift
+//  phantomgames
+//
+//  Created by Aphiwe Shozi on 2024/05/16.
+//
+
+import UIKit
+
+class GameDetailViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+    
```

Comment:
empty lines?
Please stop ignoring swift lint warnings. I'm going to start decining PR's if i see them not being addressed.